### PR TITLE
Fixing failed drone build

### PR DESCRIPTION
**Pull request by AwkwardBen**

### CODE REVIEW

**Comment by easternbloc** [comment]
self merging makes @AwkwardBen a baaaaaaaaaad boy!

**Comment by RobinKnipe** [comment]
lack of a proper (or any) description makes @AwkwardBen a baaaaaaaaaad boy!

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN groupadd -r nodejs && \
...
 WORKDIR /app
 
-ONBUILD RUN yum clean all && \
-    yum update -y -q && \
-    yum clean all && \
-    rpm --rebuilddb
+RUN yum clean all 
+RUN yum update -x 'filesystem*' -y 
+RUN yum clean all
+RUN rpm --rebuilddb
```
Comment (RobinKnipe) [inline Dockerfile:19]: Reducing layers = good boy points!

**Review by RobinKnipe — COMMENTED** [review]
Post-merge LGTM

**Pull request closed without merging**